server/@types/models/Referral.ts

```diff
@@ -40,8 +40,8 @@ type ReferralSummary = {
   earliestReleaseDate?: Person['conditionalReleaseDate'] | Person['paroleEligibilityDate'] | Person['tariffDate']
   organisationId?: Organisation['id']
   prisonerName?: {
-    firstName?: Prisoner['firstName']
-    lastName?: Prisoner['lastName']
+    firstName: Prisoner['firstName']
+    lastName: Prisoner['lastName']
```

Comment:
These two `CaseListUtils` tests need removing with this commit: ``'when `prisonerName.firstName` is `undefined`'`` and ``when `'prisonerName.lastName` is `undefined`'``

Reply:
Yeah the pacts are currently passing on the API side so I thought I'd make the API changes before merging this one in, but yep, these tests need updating

Reply:
Argh I take it back, a new deployment has randomised some data again - will need to sort this now